client/control/tests/actions/test_filterActions.js

```diff
@@ -0,0 +1,190 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import fetchMock from 'fetch-mock';
+
+import { initialState } from 'control/tests/fixtures';
+import { API_REQUEST_SETTINGS } from 'control/actions/ControlActions';
+import * as filterActions from 'control/actions/FilterActions';
+import { getFilterParamString } from 'control/selectors/FiltersSelector';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const store = mockStore({ ...initialState });
+
+describe('Filter Actions', () => {
+  afterEach(() => {
+    expect(fetchMock.calls().unmatched).toEqual([]);
+    store.clearActions();
+    fetchMock.restore();
+  });
+
+  /**
+   * loadFilters
+   * - creates an LOAD_FILTERS action
+   * - should request filters from server
+   */
+  describe('loadFilters', () => {
+    const {
+      loadFilters,
+      LOAD_FILTERS,
+    } = filterActions;
+
+    it('creates an LOAD_FILTERS action', () => {
+      const expectedAction = {
+        type: LOAD_FILTERS,
+        filters: [],
+      };
+
+      store.dispatch(loadFilters())
+      .then(() => {
+        expect(store.getActions()).toContain({ ...expectedAction });
```

Comment:
We should prefer async/await for simple cases like this.

Also, why clone the `expectedAction`? Would `expect(store.getActions()).toContain(expectedAction)` be different?